bothub/api/v2/repository/views.py

```diff
@@ -466,6 +466,13 @@ def evaluate_crossvalidation(self, request, **kwargs):
                 "users who voted for this repository",
                 type=openapi.TYPE_STRING,
             ),
+            openapi.Parameter(
```

Comment:
Esse parametro é aqui mesmo? deveria ser adicionado aqui https://github.com/bothub-it/bothub-engine/blob/3a83b14112366eb64d2d9c6c98872db3a201e876/bothub/api/v2/repository/views.py#L434